example_select_test.go

```diff
@@ -0,0 +1,75 @@
+package promptui
+
+import (
+	"fmt"
+	"strings"
+)
+
+// Any type can be given to the select's item as long as the templates properly implement the dot notation
+// to display it.
+type pepper struct {
+	Name     string
+	HeatUnit int
+	Peppers  int
+}
```

Comment:
Because this example is in its own file, and because this type is also declared in the file, this example renders differently than the others. 

To make it be the same, you could move the type declaration inside the Example func. or leave as is; up to you!

Reply:
ok! 👍